src/test/java/io/lindstrom/m3u8/parser/MasterPlaylistCommentsTest.java

```diff
@@ -0,0 +1,86 @@
+package io.lindstrom.m3u8.parser;
+
+import io.lindstrom.m3u8.model.*;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MasterPlaylistCommentsTest {
+    private final MasterPlaylistParser parser = new MasterPlaylistParser();
+
+    @Test
+    public void shouldAddCommentToMasterPlaylist() {
+        MasterPlaylist playlist = MasterPlaylist.builder()
+                .addComments(
```

Comment:
Since removing the Comment class and just using String, am I correct in understanding this change will no longer be needed as we can just add String values and won't need a builder.